setup.py

```diff
@@ -5,4 +5,4 @@
 import setuptools
 
 if __name__ == "__main__":
-    setuptools.setup()
+    setuptools.setup(include_package_data=True)
```

Comment:
put this in the `setup.cfg` under the `[options]` header. See https://cthoyt.com/2020/06/03/how-to-code-with-me-organization.html for more information on how to do packaging

Reply:
Done and tested. Works as expected!